src/Compilers/CSharp/Test/Emit2/Semantics/ExperimentalAttributeTests.cs

```diff
@@ -1368,8 +1368,8 @@ public class C<[System.Diagnostics.CodeAnalysis.Experimental("DiagID1")] T> { }
         comp.VerifyDiagnostics();
     }
 
-    [Theory, CombinatorialData]
-    public void NullDiagnosticId(bool inSource)
+    [Fact]
+    public void NullDiagnosticId()
     {
         var libSrc = """
 [System.Diagnostics.CodeAnalysis.Experimental(null)]
```

Comment:
>null [](http://example.com/codeflow?start=46&length=4)

I think it would be good to test a scenario:
- without an argument
- the argument isn't a string. An integer literal `1`, for example